index.d.ts

```diff
@@ -172,6 +172,18 @@ declare namespace RRule {
 
     function fromString(value: string): RRule;
 
+    function rrulestr(s: string, options?: RRuleStrOptions): RRule;
```

Comment:
One point: this might return an `RRule` OR an `RRuleSet`. So it might be better typed `RRule | RRuleSet`.